app/routes/repos.py

```
@@ -0,0 +1,50 @@
+router = APIRouter()
+
+
+@router.get(
+    "/{user_id}",
+    response_model=List[Dict[str, Any]],
+    status_code=status.HTTP_200_OK,
+    summary="Get all repos",
+    description="Retrieve a paginated list of repositories for a user",
+)
+async def get_repos(
+    user_id: str, pagination: PaginationParams = PaginationParams()
```

Ruff (0.11.9)

Do not perform function call `PaginationParams` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)